packages/remirror__extension-link/src/link-extension.ts

```diff
@@ -603,29 +665,84 @@ export class LinkExtension extends MarkExtension<LinkOptions> {
             }
 
             const nodeText = tr.doc.textBetween(pos, pos + node.nodeSize, undefined, ' ');
-            this.findAutoLinks(nodeText)
+
+            const foundAutoLinks: FoundAutoLink[] = [];
+
+            for (const match of findMatches(nodeText, new RegExp(/(\S)+/, 'gm'))) {
```

Comment:
Can we stored this regex in a const at the start of this file? It gives a slight performance benefit (unless things have changed) - which is valuable as this is executed quite frequently

```js
const WHITESPACE_REGEX = /\S+/gm;
```

Reply:
Good call